README.md

```diff
@@ -1,70 +1,116 @@
-# Getting Started with Create React App
+# BookShelves metrics app
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+>This App is showing few books details, like: title, author, image url, price, review and description.
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- Major languages
+  - Javascript
+  - HTML/CSS
+- Frameworks & Libraries
+  - React
+  - Redux
+- Technologies used
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+![FireShot Capture 003 - Multi Device Website Mockup Generator - techsini com](https://user-images.githubusercontent.com/80458199/195976138-60eff7e9-624d-41b2-a705-a96b24042da2.png)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo (if available)
 
-### `npm run build`
+- Deployed to Netlify. Click below link to visit.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Live Demo
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[Live Demo](https://634a5de47268306b6cf8fb28--scintillating-cupcake-d003b8.netlify.app/)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Project Video Description
 
-### `npm run eject`
+[Video Presentation](https://drive.google.com/file/d/1xD4SuzL1vCv8R33JCTENYaMD6WgHxqoh/view?usp=sharing)
```

Comment:
- [ ]  Please, let's try to keep in mind to be attentive to details. it's a skill that would make you stand out in your future job. The video is supposed to have a maximum length of 5min. It is best to use [loom](https://www.loom.com/) as this helps you not to record more than 5mins. please kindly update the video so it meets the project requirements.